components/Video/index.tsx

```diff
@@ -1,50 +1,134 @@
 "use client";
-
+import { Code } from "@/types/code";
 import Image from "next/image";
 import { useState } from "react";
 import SectionTitle from "../Common/SectionTitle";
 
-import ModalVideo from "react-modal-video";
+{/*import ModalVideo from "react-modal-video";*/}
+
+const codesData: Code[] = [
+  {
+    id: 1,
+    name: "css",
+    image: "/images/video/css-3.svg",
+  },
+  {
+    id: 2,
+    name: "dart",
+    image: "/images/video/dart.svg",
+  },
+  {
+    id: 3,
+    name: "gopher",
+
+    image: "/images/video/go-8.svg",
+  },
+  {
+    id: 4,
+    name: "html",
+    
+    image: "/images/video/html-1.svg",
+  },
+  {
+    id: 5,
+    name: "Java",
+    
+    image: "/images/video/java-14.svg",
+  },
+  {
+    id: 6,
+    name: "javascript", 
+    image: "/images/video/logo-javascript.svg",
+  },
+  {
+    id: 7,
+    name: "cmas",
+    
+    image: "/images/video/cmas.svg",
+  },
+  {
+    id: 8,
+    name: "csharp",
+    
+    image: "/images/video/csharp.svg",
+  },
+  {
+    id: 9,
+    name: "php",
+    
+    image: "/images/video/icons8-logo-php.svg",
+  },
+  {
+    id: 10,
+    name: "php",
+    
+    image: "/images/video/python-5.svg",
+  },
+  {
+    id: 11,
+    name: "react",
+    
+    image: "/images/video/react-2.svg",
+  },
+  {
+    id: 12,
+    name: "ruby",
+    
+    image: "/images/video/ruby.svg",
+  },
+  {
+    id: 13,
+    name: "rust",
+    
+    image: "/images/video/rust.svg",
+  },
+
+  {
+    id: 14,
+    name: "typescript",
+    
+    image: "/images/video/typescript-2.svg",
+  },
+  {
+    id: 15,
+    name: "swift",
+    
+    image: "/images/video/swift-15.svg",
+  }
+];
 
 const Video = () => {
```

Comment:
This video component should be renamed. Or if we are not using it, just get it delete.